### PR TITLE
DRAFT Compiler FE Support ResizeNearestNeighbor operator

### PR DESCRIPTION
**Pull request by toomuchsalt**
Add support for ResizeNearestNeighbor operator

ONE-DCO-1.0-Signed-off-by: Vladimir Plazun v.plazun@samsung.com

### CODE REVIEW

**Comment by toomuchsalt** [comment]
@seanshpark can you please take a look

**Comment by seanshpark** [comment]
Looks OK. Please post also python script too :)

File: compiler/luci/lang/include/luci/IR/Nodes/CircleResizeNearestNeighbor.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#include "luci/IR/CircleOpcode.h"
+
+#include "luci/IR/AttrFusedActFunc.h"
+#include "luci/IR/LuciNodeMixins.h"
+
+namespace luci
+{
+
+/**
+ * @brief RESHAPE in Circle
```
Comment (seanshpark) [inline compiler/luci/lang/include/luci/IR/Nodes/CircleResizeNearestNeighbor.h:30]: Comment to `RESIZE_NEAREST_NEIGHBOR`

**Comment by seanshpark** [comment]
Let's proceed with posting PRs :)

**Comment by toomuchsalt** [comment]
> Looks OK. Please post also python script too :)

Python script is already present in examples

**Comment by toomuchsalt** [comment]
Done

**Pull request closed without merging**